lib/main.js

```diff
@@ -36,7 +36,7 @@ Lambda.prototype.setup = function () {
 };
 
 Lambda.prototype.run = function (program) {
-  this._createSampleFile('event.json');
+  this._createSampleFile('event.json', program.eventFile);
```

Comment:
Sorry, didn't have much time to review your comments lately.
Thank you for merging 👍